tests/migration/migration.go

```diff
@@ -2184,7 +2184,7 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 				dv, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dv, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				libstorage.EventuallyDV(dv, 600, HaveSucceeded())
+				libstorage.EventuallyDV(dv, 600, Or(HaveSucceeded(), PendingPopulation()))
```

Comment:
Should it be `WFFC()` instead of `PendingPopulation()`?

Reply:
there are places where succeeded is expected in such case with pendingpopulation its ok to still start the vm because the dv will get populated when the vm starts. in case of waitforfirstconsumer it means we are not using populators so wffc is not enough in such cases